### ISSUE TITLE
[Bug] Running Dev server has trial expired

### ISSUE THREAD

**Issue by arcuo** [ApryseSDK/webviewer-ui#1078]
**WebViewer version**
10.12

**The current behavior**
Running the dev server for webviewer-ui shows Error modal for trial run expired

```
webviewer-core.min.js:1017 Uncaught (in promise) Error: Thank you for evaluating WebViewer! Your 7-day trial has expired.

To extend the trial, please sign-up to get a key from https://dev.apryse.com/. If you still see this error message after passing a trial key, try clearing your browser cache.
    at qa (webviewer-core.min.js:1017:372)
    at webviewer-core.min.js:1028:460
    at n (webviewer-core.min.js:66:130)
    at Object.next (webviewer-core.min.js:64:490)
    at webviewer-core.min.js:64:422
    at new Promise (<anonymous>)
    at ja (webviewer-core.min.js:64:203)
    at Object.Sna (webviewer-core.min.js:1027:529)
    at webviewer-core.min.js:762:203
    at n (webviewer-core.min.js:66:130)
```

**The expected behavior**
To be able to run the dev server or supply docs in the readme on how to add API key. Currently, we are developing on a fork.

**Steps to reproduce**
Running clean from 10.12

```
npm install --legacy-peer-deps
npm download-webviewer
npm start
```

**Comment by bollain** [comment]
hey @arcuo, 

Thanks for reaching out. 

So generally the flow for working with this repo if you choose to fork it, is to build the UI from here and then use that in your project that has the WebViewer Core files. That way you can instantiate WebViewer using the constructor, which takes a licenseKey parameter. (some users use symlinks as well)

However, I can understand that being cumbersome.

What I can suggest is to tweak this in your fork.

In this [object with the default parameters](https://github.com/ApryseSDK/webviewer-ui/blob/10.12/src/helpers/loadDocument.js#L57), add a `licenseKey` option. 
```
licenseKey: 'Put the key here',
```

You can get a key from https://dev.apryse.com/

This way while you are doing dev work you can avoid the error message - just remember to remove this once you build the UI and copy it over to your project. 

Let me know if this works out for you.

**Comment by arcuo** [comment]
Hey! I tried the fix, but in a newer version (11.1) and there I didn't have no issues with or without the fix. I've not been able to reproduce the issue since. 

Thank you for the help!
